operations/host_spawn.go

```diff
@@ -76,6 +80,22 @@ func hostCreate() cli.Command {
 				Name:  noExpireFlagName,
 				Usage: "make host never expire",
 			},
+			cli.StringSliceFlag{
+				Name:  wholeWeekdaysOffFlagName,
+				Usage: "for an unexpirable host, the days when the host should be turned off for its sleep schedule (allowed values: Sunday, Monday, Tuesday, Wednesday, Thursday, Friday, Saturday)",
+			},
+			cli.StringFlag{
+				Name:  dailyStartTimeFlagName,
+				Usage: "for an unexpirable host, the time when the host should start each day for its sleep schedule",
+			},
+			cli.StringFlag{
+				Name:  dailyStopTimeFlagName,
+				Usage: "for an unexpirable host, the time when the host should stop each day for its sleep schedule",
+			},
+			cli.StringFlag{
+				Name:  timeZoneFlagName,
```

Comment:
Can we add an example or format for timezone (both here and modify)?

Reply:
I gave an example (there's no format to this unfortunately, you have to look up your own time zone name).